src/valkey-benchmark.c

```diff
@@ -241,12 +245,12 @@ static dictType dtype = {
     NULL               /* allow to expand */
 };
 
-static valkeyContext *getValkeyContext(const char *ip, int port, const char *hostsocket) {
+static valkeyContext *getValkeyContext(const char *ip, int port, const char *hostsocket, int rdma) {
     valkeyContext *ctx = NULL;
     valkeyReply *reply = NULL;
     struct timeval tv = {0};
     if (hostsocket == NULL)
-        ctx = valkeyConnectWrapper(ip, port, tv, 0);
+        ctx = valkeyConnectWrapper(ip, port, tv, 0, rdma);
```

Comment:
We can just use `config.rdma` here, just like `config.tls` is used some lines below. This function already depends on global config. No need to add the rdma parameter to the getValkeyContext function.

The reason this function takes ip and port is for cluster connections, so they can vary, but rdma and tls are always as in the config.

```suggestion
        ctx = valkeyConnectWrapper(ip, port, tv, 0, config.rdma);
```